src/ship.ts

```diff
@@ -7,30 +7,47 @@ import {
   FPS,
   START_LIVES,
   FRICTION,
+  CVS,
+  CTX,
 } from './constants.js';
 import {fxExplode, fxThrust} from './soundsMusic.js';
-import {getCanvConsts} from './canvas.js';
-const {cvs, ctx, center} = getCanvConsts();
-let ship = {
-  x: center.x,
-  y: center.y,
-  t: 0,
-  xv: 0,
-  yv: 0,
-  r: SHIP_SIZE / 2,
-  a: (90 / 180) * Math.PI, // convert to radians
-  blinkCount: Math.ceil(SHIP_INV_DUR / SHIP_INV_BLINK_DUR),
-  blinkTime: Math.ceil(SHIP_INV_BLINK_DUR * FPS),
-  blinkOn: false,
-  canShoot: true,
-  dead: false,
-  exploding: false,
-  lives: START_LIVES,
-  lasers: [],
-  explodeTime: 0,
-  rot: 0,
-  thrusting: false,
-};
+import {laser} from './lasers.js';
+
+/**
+ *
+ */
```

Comment:
What this comments do?

Reply:
Nothing. I think I added that so vscode wouldn't complain about missing jsdocs. In the next PR, I will be migrating all docs to tsdoc and will address this there.
https://tsdoc.org/